CMake/ThirdPartyQt4.cmake

```diff
@@ -5,7 +5,7 @@
 # - preprocessor definitions needed by package
 #
 
-FIND_PACKAGE(Qt4 4.8.5 COMPONENTS QtCore QtGui QtScript Qt3Support QtXml QtOpenGL REQUIRED)
+FIND_PACKAGE(Qt4 4.8.5 COMPONENTS QtCore QtGui QtScript QtXml QtOpenGL Qt3Support REQUIRED)
```

Comment:
is the Qt3Support package still required here?

Reply:
yes, there is one last dependency: Q3ScrollView in MainWindow.*